Makefile

```diff
@@ -187,6 +187,7 @@ build : run unzipimages config elastic flush
 .PHONY : run
 run:
 	@$(MAKE) -j init getdefaultcontent db/Dockerfile
+	cp ci/scripts/duplicate-db.sh defaultcontent/duplicate-db.sh
```

Comment:
You could replace `defaultcontent` with the variable `$(CONTENT_PATH)` and extract to a Makefile rule such as:
```Make
$(CONTENT_PATH)/duplicate-db.sh: $(CONTENT_PATH)
    cp ci/scripts/duplicate-db.sh $@
```

Then set this new rule `$(CONTENT_PATH)/duplicate-db.sh` as a prerequisite for `run` and any other rules that need the file.